.github/workflows/ISSUE_TEMPLATE/---bug-report.md

```diff
@@ -0,0 +1,30 @@
+---

```

Comment:
Let's proceed with moving the templates under the `.github/ISSUE_TEMPLATE` directory, please. Just to be consistent with other projects.